packages/slice-machine/scripts/start/validateManifest.ts

```diff
@@ -53,6 +53,11 @@ See below for more info 👇`,
       return { isManifestValid: false };
     }
 
+    case ManifestState.InvalidFramework: {
+      console.log("Set framework in your config to a valid value");
```

Comment:
Why display this if we have custom errors now coming from the validator itself ?

What if we have multiple errors while validating the Manifest, don't you display them all ?